h/static/styles/vendor/icomoon.css

```diff
@@ -1,175 +1,192 @@
Y2MR4BMRY2Nz4BNy4DNTQ2Ny4BNzAWFz4BMzIWFz4BMRYGBx4BFRQOAgceARUUBhUUFjc+AzU0LgIjAgBqu4tQNF2BTBMQAWpCEicjJyYoIl0WBBILK1RCKRwZBAwVQ0oeQSEhQR5KQxUMBBkcKUNTKw4VARATTIFdNFCLu2oDs1CLu2pUmoBeGQQSCgk2IBdULB8YBwMyOwcKGCIKBRk4XkkqRRsJSTUDMQgJCQgxAzVJCRtFKkpdOBkFCzAjNEwNChIEGV9/mlRqu4tQAAAAAAMAAP/ABAADwAALAB0AKgAANyIGFRQWMzI2NTQmAxUyHgIXHgMVMzQuAiMRFTIEFhIVMzQCACQjiDhQUDg5UFDBMFxWTyIiNCMTxWq49oyrAS3hgsWh/ur+i9TRUDg4UFA4OFABk8QTIzUiIk5WXTCM9rhqAVzEg+D+0qvUAXUBFqEAAAQAAP/ABAADwAAGAA0AFAAbAAABFTMHJzM1ETUjNxcjFQcjFSc3FTMhMzUXBzUjAkCAwMCAgMDAgMDAwMDAAQDAwMDAAUDAwMDAAQDAwMDAwIDAwICAwMCAAAAAAwAA/8AEAAPAABoANQBEAAABPgExNTAOAhUUFhUUBhUUFjMyNjU0JiMiBgU+ATE1MA4CFRQWFRQGFRQWMzI2NTQmIyIGATQ2MyEyFhURJSEiJjURAUURajxIPAEBOCgoODgoBw4BOhFqPEg8AQE4KCg4OCgHDv11JRsDgBom/wD9QBomAjxjIUAIMGhgBAgDBQgEKDg4KCg4AgJjIUAIMGhgBAgDBQgEKDg4KCg4AgFCGyUmGvxAwCUbAsAAAAAAAwDA/8ADgAPAAAMABwAQAAA3IRUhASETIQU3NSEVFxUzN8ACAP4AAYABAED+gAEAQP8AQEBAAEAEAP3AwEBAQECAQAAGAEAAAAPAA4AAEAAUABgAHAAgACQAABMRFBYzITI2NRE0JiMhIgYVFyEVIRUhFSEVIRUhITMVIyczESNAJhoCwDVLJhr9ABomgAKA/YACQP3AAYD+gAJAwMBAQEADQP0AGiZLNQLAGiYmGoBAgECAQEBA/wAAAAAEAAAAHwQAA4AAOwBOAFoAZgAAATAqAiMiDgIVFB4CMzoBNw4BFRQWFyIGIyIOAhUUHgIzMj4CNTQmJy4BNTQ2Nz4BNTQmJzM3Ax4BFRQGIyImNTQ2FzIWFx4BFwMuAScmNhceARcWBiU1IxUjFTMVMzUzNQIvQVhaGS1UQScdNksuBg0GBggVEAwXDDdhRygsS2Q3QGJDIyk1EjUTISErLSpVPF4CAldsTWR2TRIhDyo5CKQ0VgkJPTQzVgkJPAHfQMDAQMADgCA2RycpRzUeAQwZDRclDgEhNUYlJDopFiE2RiQ6TiYNNBIWHhkaRysyWBEr/XEHDgc7UFQ7OVMBBgQeKiEBIwJiRERcAgFgRERercDAQMDAQAAAAAEAAP/ABAADwAAjAAABISIGFREUFjMhESM1MzU0NjsBFSMiBh0BMwcjETMyNjURNCYDVf1WR2RkRwFVgIBeQqCgDROwIJDVR2RkA8BkR/1WR2QBwIBgQl6AEw1ggP5AZEcCqkdkAAAAAQAAACAEAANgAEcAAAEOAQc+ATcOAQcuASMiDgIVFBYXLgMnDgEVFBYXLgEnMBQxFBYXDgEjIiYnHgEXDgEjIiYnHgMzMj4CNTQmNT4BNwQAHD0gITAMH0MkHFAtLEw5IQMCQXtuYCcNDzMqGjAVYUgOGw8KFAkUa0U2hUoNGQwjTFFWLJHfmE4BHzUVAv4NEQMTPCUSGgceJCE5TCwMGAwDIjlOMBg1HTdcHAEOCwJNcw4EBAICPlICKjACARYjGA1trNVoBg4HFjcgAAAAAAEAAAABgAC9zfrvXw889QALBAAAAAAA0wXCbgAAAADTBcJuAAD/qwQAA8AAAAAIAAIAAAAAAAAAAQAAA8D/wAAABAAAAP/EBAAAAQAAAAAAAAAAAAAAAAAAADoEAAAAAAAAAAAAAAACAAAABAAA1QQAAJIEAACrBAAAAAQAAGQEAAArBAAAKwQAAVUEAADVBAAA1QQAAFUEAAAABAAA1QQAAIAEAAAABAABKwQAAQAEAABrBAAAVQQAANUEAAEABAAAVQQAAIAEAABVBAABbgQAAAAEAABVBAAAkgQAANUEAAGrBAAAJgQAAAgEAAAIBAAAAwQAAFUEAACABAAA1QQAANUEAACABAAAgAQAAFUEAAErBAAAVQOAAIADxAAABAAAAAQAAAAEAAAABAAAAAQAAMAEAABABAAAAAQAAAAEAAAAAAAAAAAKABQAHgA4AEoAlADaAWQBsAIeAjICTAJmApgC0ALmAwQDLANiA3oDyAQKBCYEQgSIBLAE4ATyBUAFeAWKBaQFsAYKBpYHkAheCLgJBAkmCTQJVgl4Cb4JzAn2CigKogscC14LjAvsDA4MSgzYDQwNdAAAAAEAAAA6AK8ACAAAAAAAAgAAAAAAAAAAAAAAAAAAAAAAAAAOAK4AAQAAAAAAAQABAAAAAQAAAAAAAgAHACoAAQAAAAAAAwABACQAAQAAAAAABAABAD8AAQAAAAAABQALAAMAAQAAAAAABgABACcAAQAAAAAACgAaAEIAAwABBAkAAQACAAEAAwABBAkAAgAOADEAAwABBAkAAwACACUAAwABBAkABAACAEAAAwABBAkABQAWAA4AAwABBAkABgACACgAAwABBAkACgA0AFxoAGhWZXJzaW9uIDEuNQBWAGUAcgBzAGkAbwBuACAAMQAuADVoAGhoAGhSZWd1bGFyAFIAZQBnAHUAbABhAHJoAGhGb250IGdlbmVyYXRlZCBieSBJY29Nb29uLgBGAG8AbgB0ACAAZwBlAG4AZQByAGEAdABlAGQAIABiAHkAIABJAGMAbwBNAG8AbwBuAC4AAAADAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA') format('woff');
```

Comment:
Ah, the `update-icon-font.py` script needs to be updated so that it doesn't inline the `h.woff` file automatically. Can you cherry-pick the version of `update-icon-font.py` from bfd41e1c5ecf518b082876662e376a9ca3ed931e and use that to generate the font.